src/plugins/intel_gpu/src/graph/primitive_inst.cpp

```diff
@@ -190,6 +190,10 @@ void primitive_inst::update_shape() {
         }
         auto& dep = _node->get_dependency(i);
         auto dep_id = dep.id();
+        // should not be deps of fused prims
+        if (_node->is_fused_dep(dep_id)) {
+            continue;
```

Comment:
If i is fused dep, we can break this loop.

Reply:
done